autotst/calculator/gaussian.py

```diff
@@ -57,6 +57,8 @@ def __init__(self,
                      "mem": "5GB",
                      "nprocshared": 20,
                  },
+                 scratch=".",
+                 convergence="",
```

Comment:
Maybe add a check when initializing the calculator to assert that `convergence` is an accepted string